### ISSUE TITLE
Details on training with ResNet

### ISSUE THREAD

**Issue by SnowyJune973** [wyharveychen/CloserLookFewShot#47]
Thank you for your impressive work, and codes too! 
Given using a ResNet18 backbone leads to a ~5% raise to accuracy, as is shown in your paper, I tried to train RelationNet with a backbone of ResNet18, however the model quickly overfits and ends up to a ~60% test accuracy after 60,000 episodes of training.
I was using your code with Adam as optimizer, LR=0.001 and reduces the LR by 25% when it comes to a plateau. weight_decay=0.0005 and grad clipping=(1, L2 norm) were also performed.

I wonder where's the difference between my setting and yours, and how you achieved the 70% accuracy in ResNet18+RelationNet. Thanks a lot!

**Comment by yangt1013** [comment]
Hello,I haven't changed the parameters. The network precision is 76,releationnet+resnet18

**Comment by smiler96** [comment]
> Hello,I haven't changed the parameters. The network precision is 76,releationnet+resnet18

hello,  i train renet10+protnet  while it overfits after 10k episodes, have you met this? no data augmentation in training stage.

**Comment by weitliu** [comment]
@yangt1013 Thanks for your info. However, the results on the paper is showing 69.83±0.68 in the last row of Table A5. Can you please confirm?
